Source/WebKit/CMakeLists.txt

```diff
@@ -235,6 +236,7 @@ set(WebKit_MESSAGES_IN_FILES
 
     Shared/Notifications/NotificationManagerMessageHandler
 
+    UIProcess/DigitalCredentials
```

Comment:
Should likely be:
```
UIProcess/DigitalCredentials/DigitalCredentialsCoordinatorProxy
```

Likely why it's not building for glib ports.

Reply:
ah